### PR TITLE
Fixing links

### PR DESCRIPTION
**Pull request by CsatariGergely**

### CODE REVIEW

File: content/en/docs/glossary.md [hunk 1]
```diff
@@ -31,8 +34,18 @@ is part of the "Kubernetes Everywhere" principle).
...
+services, applications, etc.) which:
+
+* makes configuration data the source of truth, stored separately from the live state
+* uses a uniform, serializable data model to represent configuration
+* separates code that acts on the configuration from the data and from packages / bundles of the data
+* abstracts configuration file structure and storage from operations that act upon the configuration data; clients
+  manipulating configuration data don’t need to directly interact with storage (git, container images)
+
+Source of definition and more information about Configuration as Data can be found in the
+[kpt documentation](https://github.com/kptdev/kpt/blob/main/docs/design-docs/06-config-as-data.md#configuration-as-data-1).
```
Comment (efiacor) [inline content/en/docs/glossary.md:48]: I know the porch docs are not "ready" but should we link to them instead?
https://github.com/nephio-project/porch/blob/main/docs/06-config-as-data.md

File: content/en/docs/guides/install-guides/package-transformations.md [hunk 2]
```diff
@@ -369,6 +369,12 @@ The new cluster comes up after a number of minutes.
 
 # Transformations in the Workload cluster creation
 
+Test mest
```
Comment (efiacor) [inline content/en/docs/guides/install-guides/package-transformations.md:372]: Just a filler/placeholder?

File: content/en/docs/glossary.md (hunk 1 above)
Comment (CsatariGergely) [reply content/en/docs/glossary.md:48]: Oh, yes, definitely. For me it is not really clear where KPT ends and where Porch starts in the documentation. 
On the mid term I would like to move the Porch documentation under the Nephio docs (https://github.com/nephio-project/nephio/issues/744). On long term the Porch documentation should follow the code of Porch+

File: content/en/docs/guides/install-guides/package-transformations.md (hunk 2 above)
Comment (CsatariGergely) [reply content/en/docs/guides/install-guides/package-transformations.md:372]: Oh, yes, thanks for catching this. I've realized that Docsy does not render the H1 headings to the right side page content list. This is remained after the debugging of the issue. I will try to figure out how to deal with H1 headings and create a PR to fix them in the whole docs set.

File: content/en/docs/glossary.md (hunk 1 above)
Comment (CsatariGergely) [reply content/en/docs/glossary.md]: Fixed in [d698826](https://github.com/nephio-project/docs/pull/130/commits/d698826176388387694ec439d1a3bc3b5d636a0e)

File: content/en/docs/guides/install-guides/package-transformations.md (hunk 2 above)
Comment (CsatariGergely) [reply content/en/docs/guides/install-guides/package-transformations.md]: Fixed in [d698826](https://github.com/nephio-project/docs/pull/130/commits/d698826176388387694ec439d1a3bc3b5d636a0e)

**Comment by arora-sagar** [comment]
@efiacor Can we merge this MR? I suppose it will fix the errors in other MRs.

**Review by liamfallon — COMMENTED** [review]
Thanks for fixing the links.

**Pull request merged**